examples/nextjs-test/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@examples/nextjs-test",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "concurrently --kill-others \"pnpm dev:next\" \"wait-on http://localhost:3000 && pnpm dev:trigger\"",
+    "dev:next": "next dev",
+    "dev:trigger": "pnpm dlx @trigger.dev/cli@latest dev",
```

Comment:
This should use the locally built cli, just add `@trigger.dev/cli` to the dev dependencies like so:

```json
  "@trigger.dev/cli": "workspace:*"
```

And then update this script to `trigger-cli dev`

Will just need to add a `pnpm run build --filter @trigger.dev/cli` step to the e2e tests

Reply:
I've updated this script to run `trigger-cli dev` directly.